### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by debragordon**
Just started the project ... view at your own risk!

### CODE REVIEW

**Comment by debragordon** [comment]
I believe the quiz is ready for review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+<body>
+    <div class="page-wrap">
+    <article class="product">
+      <header>
+        <h1>Black Widow Golf Cleats</h1>
+      </header>
+      <div class="container">
+        <section class="details">
+          <img src="/images/BLACK_BW_FT.png" alt="">
+  <!--         <h3>Product Description</h3> -->
```
Comment (zoeames) [inline index.html]: go ahead and delete this line.  As a general rule of thumb comments that describe your code are good.  anytime you comment out code you can go ahead and just delete it.

**Comment by zoeames** [comment]
Really nice first quiz!  You nailed it.  The only thing I would say is go back and delete the code you have commented out.  Its no longer needed.  once you do that you can merge this branch.

👍

**Comment by debragordon** [comment]
ok, I will go into this file on my computer, delete the lines, add and merge to master and call this one donezo! 

THANKS!!

**Pull request merged**